scss/grid/_grid.scss

```diff
@@ -30,6 +30,11 @@ $grid-column-align-edge: true !default;
 /// @type Number
 $block-grid-max: 8 !default;
 
+/// Selector used for an alias of column (with @extend). If `false`, no alias is created.
+/// @since 6.4.0
```

Comment:
I agree with @IllusionPerdu:
- We can not assume the version number of a commit. We can use versions inside the code only to reference _previous versions_ (already released) or _planned changes_ (not in the current commit).
- `@since` does not really help folks that are using a previous version.